doc/source/news.rst

```diff
@@ -5,6 +5,52 @@
 News
 ====
 
+.. _release-12-0-9:
+
+Release 12.0.9 - 2022-10-28
+---------------------------
+
+Improvements
+------------
+
+* [:doc:`reference/functions/escalate`] Added a document for the ``escalate()`` function.
+
+* [:doc:`reference/normalizers`] Added NormalizerHTML. (Experimental)
+
+  This is a normalizer for HTML.
+
+  Currently this supports only unescaping reference characters like ``&amp;`` or ``&#38;``.
+
+  Here is a sample query for NormalizerHTML.
+
+  .. code-block::
+  
+     normalize NormalizerHTML "Groonga &amp; Mroonga &#38; Rroonga"
+     [[0,1666854852.063215,0.0006151199340820312],{"normalized":"Groonga & Mroonga & Rroonga","types":[],"checks":[]}]
+  
+  In this sample, ``&amp;`` and ``&#38;`` are unescaped to ``&``.
+
+* [httpd] Updated bundled nginx to 1.23.2.
+
+  Contains security fix of CVE-2022-41741 and CVE-2022-41742.
+
+  Please refer to https://nginx.org/en/CHANGES about the changes of nginx 1.23.2. 
+
+* Suppressed logging a lot of same messages when no memory available.
+
+  Groonga could log a lot of ``mmap failed!!!!`` when no memory available.
+  We fixed to log the above message as less duplicates as possible.
+
+Fixes
+-----
+
+* [:doc:`reference/commands/select`] Fixed a bug that Groonga could crash or return incorrect results when specifying :ref:`select-n-workers`.
+
+  This had occured when using :ref:`select-n-workers` with a value greater than ``1`` and ``drilldowns[{LABEL}].filter`` at the same time.
```

Comment:
```suggestion
  This crash had occurred when using :ref:`select-n-workers` with a value greater than ``1`` and ``drilldowns[{LABEL}].filter`` at the same time.
```
Unless you specify what is this, I could read ”Fixed a bug had occurred....."
( occured is Occurred)